server/api/login_handler.py

```diff
@@ -24,9 +24,11 @@ def post(self):
         user = User.authenticate(email, password)
 
         if user:
+            token = create_refresh_token(user.to_dict())
```

Comment:
Token does not need to contain entire user object. it only needs to have user id.  all other fields are unnecessary.
`create_refresh_token(user.to_dict())` should be changed to `create_refresh_token({"id": user.id})`